templates/header-user.cfg.xml.erb

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://wiki.openrepose.org/display/REPOSE/Header+Identity -->
+<header-user xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+                 xmlns='http://docs.openrepose.org/repose/header-user/v1.0'
+                 xsi:schemaLocation='http://docs.openrepose.org/repose/header-user/v1.0 ../config/header-identity-configuration.xsd'>
+
+<%- if @source_headers -%>
+    <source-headers>
+      <%- @source_headers.each do |header| -%>
+        <header <% header.sort.map do |name, value| %>id="<%= name %>" quality="<%= value %>" <% end %>/>
```

Comment:
If I am reading this correctly, the template would render like this:
```
    <source-headers>
        <header id="header1" quality="quality1" id="header2" quality="quality2" id="header3" quality="quality3" />
    </source-headers>
```

I'd assume that it should look like this:
```
    <source-headers>
        <header id="header1" quality="quality1" />
        <header id="header2" quality="quality2" />
        <header id="header3" quality="quality3" />
    </source-headers>

```

Reply:
on line 10, i grab each header individually, the logic on line 11 simply separates out the name and value for that one header. so this should still produce the correct results (i believe my test verifies this)

I pulled from here, for example: https://github.com/rackerlabs/puppet-repose/blob/master/templates/uri-normalization.cfg.xml.erb#L37

Reply:
ah yeah. you are right